apps/desktop/desktop_native/bitwarden_chromium_importer/Cargo.toml

```
@@ -0,0 +1,34 @@
+[package]
+name = "bitwarden_chromium_importer"
+edition = { workspace = true }
+license = { workspace = true }
+version = { workspace = true }
+publish = { workspace = true }
+
+[dependencies]
+aes = { workspace = true }
+aes-gcm = "=0.10.3"
+anyhow = { workspace = true }
+async-trait = "=0.1.88"
+base64 = { workspace = true }
+cbc = { workspace = true, features = ["alloc"] }
+hex = { workspace = true }
+homedir = { workspace = true }
+log = { workspace = true }
+pbkdf2 = "=0.12.2"
+rand = { workspace = true }
+rusqlite = { version = "=0.35.0", features = ["bundled"] }
+serde = { workspace = true, features = ["derive"] }
+serde_json = { workspace = true }
+sha1 = "=0.10.6"
+
+[target.'cfg(target_os = "macos")'.dependencies]
+security-framework = { workspace = true }
+
+[target.'cfg(target_os = "windows")'.dependencies]
+tokio = { workspace = true, features = ["full"] }
+winapi = { version = "=0.3.9", features = ["dpapi", "memoryapi"] }
+windows = { workspace = true, features = ["Win32_Security", "Win32_Security_Cryptography", "Win32_Storage_FileSystem", "Win32_System_IO", "Win32_System_Memory", "Win32_System_Pipes", "Win32_System_Services", "Win32_System_Threading", "Win32_UI_Shell", "Win32_UI_WindowsAndMessaging"] }
+
+[target.'cfg(target_os = "linux")'.dependencies]
+oo7 = { workspace = true }
```

We should enable the more strict linting on all new crates.

```suggestion
oo7 = { workspace = true }

[lints]
workspace = true

```